### ISSUE TITLE
[BUG] vs code seems not hitting tailwind.config.js using turborepo

### ISSUE THREAD

**Issue by thomas-guehenneux** [francoismassart/eslint-plugin-tailwindcss#236]
Hello.

Using Turborepo, vs code displays false positives `Classname '{classnanme}' is not a Tailwind CSS class!` warning.  Linting has no issue and I did not have the same issue on Webstorm. I guess that the issue is tailwind.config.js is not in the root of the project in monorepo. 

<img width="819" alt="Screen Shot 2023-04-09 at 0 46 48" src="https://user-images.githubusercontent.com/97498481/230730921-5a33d994-117f-4f4a-ac3f-b316d277d7a0.png">

The issue can be reproduced using this repository: https://github.com/thomas-guehenneux/example_turborepo_issue_eslint_tailwind

**Comment by joepetrillo** [comment]
Add the config option - I think this fixed it for me!

Example
```js
  "rules": {
    "tailwindcss/no-custom-classname": [
      "warn",
      {
        "config": "apps/frontend/tailwind.config.js"
      }
    ]
  },
```

**Comment by thomas-guehenneux** [comment]
The issue is that if I do something like that, eslint in apps/web is failing because it does not find tailwind.config.js. I guess it searches for  apps/web/tailwind.config.js starting from app/web/. directory.

**Comment by francoismassart** [comment]
@thomas-guehenneux maybe you can try to temporary using an absolute path...
